api/oci/extensions/repositories/docker/docker_test.go

```diff
@@ -0,0 +1,20 @@
+//go:build docker_test
```

Comment:
when and where is this test executed?

Reply:
you can execute it locally. Docker is not necessarily available when executing the standard tests.
The same has been done for the redis pubsub extension.

Reply:
okay, do we have an overview which additional integration tests are available?
are those already executed in https://github.com/open-component-model/ocm-integrationtest/ ?